src/schema/schema_validators.py

```diff
@@ -579,13 +579,21 @@ def validate_publication_date(property_key, normalized_entity_type, request, exi
     The json data in request body, already after the regular validations
 """
 def validate_creation_action(property_key, normalized_entity_type, request, existing_data_dict, new_data_dict):
-    accepted_creation_action_values = ["central process", "lab process"]
+    accepted_creation_action_values = ["central process", "lab process", "external process"]
     creation_action = new_data_dict[property_key].lower()
     if creation_action and creation_action not in accepted_creation_action_values:
         raise ValueError("Invalid {} value. Accepted values are: {}".format(property_key, ", ".join(accepted_creation_action_values)))
     if creation_action == '':
         raise ValueError(f"The property {property_key} cannot be empty, when specified.")
-
+    if creation_action == 'external process':
+        if normalized_entity_type.lower() != "dataset":
```

Comment:
This validator is specific to Dataset, and Publication already rejects the `creation_action` in request payload at the schema level. This check becomes unnecessary and will never satisfy the condition.